web/src/app/lists/PaginatedList.tsx

```diff
@@ -197,13 +197,14 @@ export function PaginatedList(props: PaginatedListProps): JSX.Element {
 
     // must be explicitly set when using, in accordance with TS definitions
     const urlProps = item.url && {
-      component: function ListAppLink(props: AppLinkProps) {
-        return (
-          <li>
-            <AppLink {...props} />
+      component: React.forwardRef<HTMLLIElement, AppLinkProps>(
+        ({ children, ...props }, ref) => (
+          <li ref={ref}>
+            <AppLink {...props}>{children}</AppLink>
```

Comment:
Doesn't `ref` need to be passed to AppLink (which has `forwardRef` internally) to pass it to `Link`?